docs/features/migrate_to_pootle_fs.rst

```diff
@@ -38,6 +38,31 @@ files on disk will likely mess with direct Pootle FS change detection.
 You may want to look at the format adaptors for future massaging or formats.
 
 
+.. _migrate_to_pootle_fs#replacing-update_stores-and-sync_stores:
+
+Replacing update_stores and sync_stores
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:command:`update_stores` can be replaced with the following set of Pootle FS
```

Comment:
Please always use `:djadmo:` for Pootle and Django commands, this automatically links to them in the docs.  `:command:` is just styling and for commands like `git`, etc

Reply:
I used `:command:` on purpose for those commands that are deprecated, but will change to `:djadmin:`.